### PR TITLE
Fix catalog sync error using /items_batch for additional_variant_attributes

### PR DESCRIPTION
**Pull request by denniskwon**
Update field name and format for additional_variant_attribute to resolve Facebook catalog sync for variable products.

### CODE REVIEW

File: includes/fbproduct.php [hunk 1]
```diff
@@ -605,7 +605,7 @@ public function prepare_product( $retailer_id = null, $type_to_prepare_for = sel
 				$product_data                            = $this->apply_enhanced_catalog_fields_from_attributes( $product_data, $google_product_category );
 			}
 
-			// add the Commerce values (only inventory for the momenbt)
+			// add the Commerce values (only inventory for the moment)
```
Comment (maxdupenois) [inline includes/fbproduct.php:608]: lol

**Review by maxdupenois — APPROVED** [review]
LGTM

File: changelog.txt [hunk 2]
```diff
@@ -1,5 +1,8 @@
 *** Facebook for WooCommerce Changelog ***
 
+2020.11.18 - version 2.1.5
```
Comment (wvega) [inline changelog.txt:3]:
```suggestion
2020.nn.nn - version 2.1.5-dev.1

```

File: facebook-for-woocommerce.php [hunk 3]
```diff
@@ -10,7 +10,7 @@
  * Description: Grow your business on Facebook! Use this official plugin to help sell more of your products using Facebook. After completing the setup, you'll be ready to create ads that promote your products and you can also create a shop section on your Page where customers can browse your products on Facebook.
  * Author: Facebook
  * Author URI: https://www.facebook.com/
- * Version: 2.1.4
+ * Version: 2.1.5
```
Comment (wvega) [inline facebook-for-woocommerce.php:13]:
```suggestion
 * Version: 2.1.5-dev.1
```

File: readme.txt [hunk 4]
```diff
@@ -3,7 +3,7 @@ Contributors: facebook, automattic, woothemes
 Tags: facebook, shop, catalog, advertise, pixel, product
 Requires at least: 4.4
 Tested up to: 5.5.3
-Stable tag: 2.1.4
+Stable tag: 2.1.5
```
Comment (wvega) [inline readme.txt:6]:
```suggestion
Stable tag: 2.1.5-dev.1
```

File: readme.txt [hunk 5]
```diff
@@ -39,6 +39,9 @@ When opening a bug on GitHub, please give us as many details as possible.
 
 == Changelog ==
 
+= 2020.11.18 - version 2.1.5
```
Comment (wvega) [inline readme.txt:42]:
```suggestion
= 2020.nn.nn - version 2.1.5-dev.1
```

**Review by wvega — COMMENTED** [review]
@denniskwon thank you for putting this together! This update removes the warnings returned by the `items_batch` endpoint for me as well. 

I noticed that the products still don't show variants information on the Catalog: https://cloud.skyver.ge/d5uPrRgB. Do you think that can cause issues for product visualization on the shop?

I left a couple of suggestions to match what the deploy tool we use expects: a placeholder date `2020.nn.nn` and a dev version `2.1.5-dev.1`.

File: includes/fbproduct.php (hunk 1 above)
Comment (denniskwon) [reply includes/fbproduct.php:608]: i got your back. :)

**Pull request merged**